helper/builtinplugins/registry_min.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+//go:build minimal
+
+package builtinplugins
+
+import (
+	credJWT "github.com/hashicorp/vault-plugin-auth-jwt"
+	logicalKv "github.com/hashicorp/vault-plugin-secrets-kv"
+	credAppRole "github.com/hashicorp/vault/builtin/credential/approle"
+	credCert "github.com/hashicorp/vault/builtin/credential/cert"
+	credUserpass "github.com/hashicorp/vault/builtin/credential/userpass"
+	logicalPki "github.com/hashicorp/vault/builtin/logical/pki"
+	logicalSsh "github.com/hashicorp/vault/builtin/logical/ssh"
+	logicalTransit "github.com/hashicorp/vault/builtin/logical/transit"
+)
+
+func newRegistry() *registry {
```

Comment:
I was debating about the two approaches. The extension approach as it required for loop extension functions each backend/plugin since go map doesn't `append()` like slices. Displaying clearly what is each Vault is simple enough to not need tests.

However, I could see my approach will become a toil at some point when a new bundled plugin is added or if we later offer more SKUs for Vault+bundled plugins.

Reply:
TIL, `maps.Copy()` can replace the registry extension for loop.